app/config.py

```diff
@@ -453,11 +453,24 @@ class Development(Config):
 
 class Decoupled(Development):
     NOTIFY_ENVIRONMENT = "decoupled"
-    ADMIN_BASE_URL = "http://admin:6012"
+    ADMIN_BASE_URL = "http://admin.ecs.local:6012"
     SQLALCHEMY_DATABASE_URI = "postgresql://pg/emergency_alerts"
-    REDIS_URL = "redis://api:6379/0"
-    API_HOST_NAME = "http://api:6011"
-    TEMPLATE_PREVIEW_API_HOST = "http://api:6013"
+    REDIS_URL = "redis://api.ecs.local:6379/0"
```

Comment:
We're running redis ourselves rather than using elasticache?

Reply:
I believe Redis was formerly used as a backend to Celery, so store task state and responses, primarily to surface this data to a dashboard or other monitoring service.  We've left it in place to reduce the risk of breaking changes - the validity of including Redis will be addressed when we start removing legacy Notify code/dependencies.